docs/reference/builder.md

```diff
@@ -1396,6 +1396,31 @@ To use these, simply pass them on the command line using the flag:
 --build-arg <varname>=<value>
 ```
 
+These pre-defined variables are exluded from the output of `docker history` by default.
+This reduces the risk of accidentally leaking sensitive authentication information
+in an `HTTP_PROXY` variable. This behaviour can be overriden by providing an
```

Comment:
Please use US spelling of "behavior". :) Also prefer active voice "To override this behavior, provide an explicit...."